app/views/spree/admin/orders/_form.html.haml

```diff
@@ -9,7 +9,7 @@
 
   - if order.line_items.exists?
     %fieldset#order-total.no-border-bottom{"data-hook" => "order_details_total"}
-      %legend= t(".order_total")
+      %legend{ align: 'center' }= t(".order_total")
```

Comment:
you can use the class .text-center instead, I think it will work, can you please give it a try?

` %legend.text-align= t(".order_total")`

Reply:
> you can use the class .text-center instead, I think it will work, can you please give it a try?
> 
> ` %legend.text-align= t(".order_total")`

Going to try this solution.

Reply:
It did not work with text align.
The adjustment with align center, the total with text-align class below.
![image](https://user-images.githubusercontent.com/3486644/97768366-23b27480-1b55-11eb-9b37-8b25154828df.png)

Reply:
ok, thanks for trying it out anyway! :-)